tty/pc-tty/ttypc_vga.c

```diff
@@ -257,9 +330,23 @@ void _ttypc_vga_getcursor(ttypc_vt_t *vt)
 }
 
 
+#define INVERT_COLORS(val) (((val & 0x0f00) << 4) | ((val & 0xf000) >> 4) | (val & 0xff))
+
+
 void _ttypc_vga_setcursor(ttypc_vt_t *vt)
 {
+	static int curr_pos = -1;
```

Comment:
Consider adding this to the `vt` - whole context is held by this structure, this static breaks this rule and makes the whole code non-stateless